### ISSUE TITLE
Bug: combobox-item height should match dropdown-item height (24/32/44)

### ISSUE THREAD

**Issue by caripizza** [Esri/calcite-components#3144]
### Actual Behavior
calcite-combobox-item non-wrapping text heights are 32/40/52 but should match calcite-dropdown-item heights and be 24px for small, 32px for medium, and 44px high for large scale items.

### Expected Behavior
A small combobox-item total height is 24px, a medium total height is 32px, and a large total item height is 44px.

### Reproduction Steps and Sample
Each scale of combobox-item with dropdown-item beside it is illustrated here, showing the height diffs: https://codepen.io/caripizza-the-animator/pen/oNwVPrx

_Sample_:

1. `npm start` on master > calcite-combobox.html > notice the item heights are larger than 24/32/44

### Relevant Info

_Version_: `@esri/calcite-components@1.0.0-beta.66`

- [x] CDN
- [ ] NPM package

**Comment by caripizza** [comment]
@macandcheese should we add combobox-item to the "Combobox" Figma page so it matches Dropdown?

**Comment by macandcheese** [comment]
Yeah, there should probably be versions in Figma of the various indentation levels and states for combobox items.
